app/src/main/java/cm/aptoide/pt/toolbox/ToolboxContentProvider.java

```diff
@@ -62,11 +64,13 @@
     uriMatcher = new UriMatcher(UriMatcher.NO_MATCH);
     uriMatcher.addURI(BuildConfig.CONTENT_AUTHORITY, "token", TOKEN);
     uriMatcher.addURI(BuildConfig.CONTENT_AUTHORITY, "refreshToken", REFRESH_TOKEN);
-    uriMatcher.addURI(BuildConfig.CONTENT_AUTHORITY, "repo", REPO);
+    uriMatcher.addURI(BuildConfig.CONTENT_AUTHORITY, "storeName", STORE_NAME);
```

Comment:
warning: by doing this change you will break older versions of uploader and also other potential clients that use this (backup-apps, toolbox)
Because we don't have a way to version this right now in place, it is better to only make additions

Reply:
I only changed it because of this comment https://github.com/Aptoide/aptoide-uploader/pull/37#discussion_r480980865
I can revert it again, the main reason i kept the same names as before was to avoid problems, but based on the comment I assumed I supposed to change it anyway